pxtblocks/blocklyloader.ts

```diff
@@ -11,7 +11,9 @@ namespace pxt.blocks {
         images: 45,
         variables: 330,
         text: 160,
-        lists: 260
+        lists: 260,
+	math: 230,
```

Comment:
I did not know that. I will try it.

I have just added a commit to replace tabs with spaces.